src/Infra/MySQL/MySQLItemRepository.ts

```diff
@@ -1,14 +1,21 @@
-import { injectable } from 'inversify'
+import { inject, injectable } from 'inversify'
 import { EntityRepository, Repository } from 'typeorm'
 import { ContentType } from '@standardnotes/common'
 import { Item } from '../../Domain/Item/Item'
 import { ItemQuery } from '../../Domain/Item/ItemQuery'
 import { ItemRepositoryInterface } from '../../Domain/Item/ItemRepositoryInterface'
 import { Timer } from '@standardnotes/time'
+import TYPES from '../../Bootstrap/Types'
 
 @injectable()
 @EntityRepository(Item)
 export class MySQLItemRepository extends Repository<Item> implements ItemRepositoryInterface {
+  constructor(
+    @inject(TYPES.Timer) private timer: Timer,
```

Comment:
I've tried injecting dependencies into the extender of `Repository<T>` and it's a bit tricky - I'm pretty sure this will fail - have you tried this locally?